app/controllers/transcriptions_controller.rb

```diff
@@ -67,11 +68,19 @@ def type_invalid?
     params[:data][:type] != "transcriptions"
   end
 
+  def whitelisted_attributes?
+    params[:data][:attributes] && params[:data][:attributes].keys.all? { |key| whitelist.include? key }
+  end
+
   def approve?
     update_attrs["status"] == "approved"
   end
 
   def allowed_filters
     [:id, :workflow_id, :group_id, :flagged, :status]
   end
+
+  def whitelist
```

Comment:
nonblocking: can this be a little more descriptive? maybe `update_attr_whitelist`

Reply:
Of course